src/main/resources/application.properties

```diff
@@ -1,3 +1,4 @@
+app.member.hosted-domain=ucsb.edu
```

Comment:
Not sure if this is supposed to be passed alongside the rest of the code.

Reply:
Correct, this line was not meant to be pushed